README.md

```diff
@@ -45,7 +45,7 @@ The common options are:
 | `maxRetries`  | The maximum number of retries to be made when a request fails. Default: 10                                                                                                                                                                                                                                        |                                            |
 | `limit`       | The number of items to fetch for this resource type to create the DataFrame. Note that this is different from the SQL `SELECT * FROM ... LIMIT 1000` limit. This option specifies the limit for items to fetch from CDF, *before* filtering and other transformations are applied to limit the number of results. |                                            |
 | `batchSize`   | The maximum number of items to read/write per API call.                                                                                                                                                                                                                                                           |                                            |
-| `baseUrl`     | The prefix for all CDF API calls. Default: https://api.cognitedata.com                                                                                                                                                                                                                                            |                                            |
+
```

Comment:
Do we have an issue to add support for this again? The Scala SDK now supports it, so it should be easy.

Reply:
We do now! https://github.com/cognitedata/cdp-spark-datasource/issues/315